helpers/users.go

```diff
@@ -0,0 +1,17 @@
+package helper
+
+import (
+	"golang.org/x/crypto/bcrypt"
+)
+
+func EncryptPassword(password string) (string, error) {
+	encryptedPassword, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
+	if err != nil {
+		return "", err
+	}
+	return string(encryptedPassword), nil
```

Comment:
For me, the spacing would be great here
```suggestion
	}

	return string(encryptedPassword), nil
```